assets/src/styles/blocks/core-overrides/Heading.scss

```diff
@@ -0,0 +1,18 @@
+.is-style-chevron::after {
```

Comment:
I see that the chevron style is used in other parts of the code, so, don't we need to convert it as a `mixing` or a shared class. I mean not in this file.

Reply:
Or a function and then you can pass the `width` and `height` as a parameters

Reply:
That sounds good!